src/ConnectionManager.h

```diff
@@ -18,9 +18,7 @@
 #ifndef CONNECTION_MANAGER_H_INCLUDED
 #define CONNECTION_MANAGER_H_INCLUDED
 
-#ifndef ARDUINO_CLOUD_DEBUG_LEVEL
```

Comment:
I'd leave the guard to allow multiple inclusion of this file

Reply:
@facchinm this is only to define a default in case it wasn't defined in the sketch, but it's definitely not needed anymore.
there's still a check against multiple inclusion